### ISSUE TITLE
Dotplot ignoring arguments

### ISSUE THREAD

**Issue by NicolasSompairac** [zqfang/GSEApy#195]
Hello there,

Using the latest version of gseapy (1.0.4), after trying to generate a figure with dotplot, I do not manage to make it show the top n terms from the different gene sets. The dataframe from the enrichment does contain several gene sets and the adjusted p-values are significant enough to pass the cutoff.
But even by specifying the "x" argument as "Gene_set", it seems that it is totally ignored and the plot shows only the "Combined Score".

**Comment by NicolasSompairac** [comment]
By the way, same thing happens with the "y" argument as well for some reason. Setting it to "Overlap" for example still generates the default "Term" values.

**Comment by zqfang** [comment]
Sorry for the bad experience that you've had with gseapy. It is possible that the pandas version changed ? It's the same code either install use pip or bioconda for  v1.0.4.

I'll go through the code when I have time

**Comment by NicolasSompairac** [comment]
Hi @zqfang , thanks for taking notice of the issue :)
Honestly, you don't have to be sorry, especially since I managed to make it work in the end.

I am however confused about the reason of the issue since the plot itself was generated, there were no argument issues so they did exist in the dotplot function but the reason of why it constantly reverted to default arguments is just unclear.
Regarding your comment, I did not change the pandas version afaik and a simple `pip uninstall gseapy` followed by an install did the trick.
I unfortunately did not save the logs of the `pip install gseapy` result but I clearly remember that it didn't update or reinstall some dependencies and stated that they were already present.
